### PR TITLE
Quote1

### PR DESCRIPTION
**Pull request by aqk**

### CODE REVIEW

**Comment by aqk** [comment]
Note that this won't pass tests without https://github.com/Chia-Network/clvm_tools/pull/20

File: clvm/operators.py [hunk 1]
```diff
@@ -8,8 +8,9 @@
 from .casts import int_to_bytes
 from .op_utils import operators_for_module
 
+QUOTE_KW = "quote"
```
Comment (arvidn) [inline clvm/operators.py:11]: we're changing it from `q` to `quote`?

Comment (arvidn) [reply clvm/operators.py:11]: I assume this is to ensure there are no `q`s accidentally lurking anywhere that we forgot to change. The down-side with this is that it makes this patch touch everything, and it makes it harder to interleave into the commit-stream

Comment (aqk) [reply clvm/operators.py:11]: Yes, I think this was the reasoning behind Richard asking for this change.

Comment (aqk) [reply clvm/operators.py:11]: Just talked with Richard. Please disregard this PR until the next update. Thanks!

**Pull request closed without merging**